docker/build.sh

```diff
@@ -151,13 +151,13 @@ function make_jar()
     GIT_HASH=$(git log -n 1 --pretty=format:%h)
 
     # Make jar & cache sbt dependencies.
-    JAR_CMD="$(docker run --rm --link $DB_CONTAINER:mysql \
+    docker run --rm --link $DB_CONTAINER:mysql \
```

Comment:
This makes all build output display in Jenkins `leonardo-build`.

…although Jenkins crashed right after I did this 😰